examples/torque_activations_driven/align_markers_torque_act.py

```diff
@@ -0,0 +1,78 @@
+import biorbd
+
+from biorbd_optim import (
+    Instant,
+    OptimalControlProgram,
+    ProblemType,
+    Objective,
+    Constraint,
+    Bounds,
+    QAndQDotBounds,
+    InitialConditions,
+    ShowResult,
+    OdeSolver,
+)
+
+
+def prepare_ocp(biorbd_model_path, number_shooting_points, final_time, ode_solver=OdeSolver.RK):
+    # --- Options --- #
+    # Model path
+    biorbd_model = biorbd.Model(biorbd_model_path)
+
+    # Problem parameters
+    torque_activation_min, torque_activation_max, torque_activation_init = -1, 1, 0
+
+    # Add objective functions
+    objective_functions = {"type": Objective.Lagrange.MINIMIZE_TORQUE, "weight": 100}
+
+    # Dynamics
+    problem_type = ProblemType.torque_activations_driven
```

Comment:
Could you confirm that the only difference between this file and align_markers.py is this line (and eventually the `torque_activation_min, torque_activation_max, torque_activation_init = -1, 1, 0`? If so, I think it would make sense to put these two lines in the align_markers.py file and add a selector in the parameter (with the default value being the previous behaviour)